values/test/0chain-values.yaml

```diff
@@ -535,7 +535,7 @@ sharder:
       # % of fees and rewards for generator
       serviceCharge: "0.10 "     # [0; 1) of all fees
       # max number of delegate pools allowed by a node in miner SC
-      numberOfDelegates: "10"   # max number of delegate pools
+      numberOfDelegates: "15"   # max number of delegate pools
```

Comment:
This file isn't currently read by the system tests helm command so i don't think this will take effect 
to fix either pass in the single override number_of_delegates to the 0chain install command or send in the whole file 
-f values/test/0chain-values.yaml

Reply:
Done using `--set` @stewartie4 
![image](https://user-images.githubusercontent.com/42718091/164387560-8e558db9-3e26-4901-a9fc-79b56315f046.png)